tenets/codelingo/psr1/camel-case-method-name/expected.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+   "Comment": "Method names MUST be declared in camelCase",
+   "Filename": "example.php",
+   "Line": 7,
+   "Snippet": "  }\n\n  function HelloOtherWorld() {\n    print \"another hello\";\n  }"
```

Comment:
Line 7 is still in camel case, it just starts with an upper cased letter. I know PSR isn't that detail though. Here's a comment that goes into a little detail on naming convention https://softwareengineering.stackexchange.com/a/149321. They do suggest methods should be camel case starting with lower case.